src/couch_mrview/src/couch_mrview_index.erl

```diff
@@ -264,6 +267,29 @@ get_ddoc(DbName, DesignDocs, DDocId) ->
     end).
 
 
+set_partitioned(Db, State) ->
+    #mrst{
+        design_opts = DesignOpts
+    } = State,
+    DbPartitioned = couch_db:is_partitioned(Db),
+    ViewPartitioned = proplists:get_value(<<"partitioned">>, DesignOpts),
```

Comment:
I think we should do `ViewPartitioned = proplists:get_value(<<"partitioned">>, DesignOpts, DbPartitioned),` here. That would reduce two of the case statement lines since if there is no option then the view partition option is the same as the `DbPartitioned` value.

Reply:
Also when should we use `couch_util:get_value` and when should we use `proplists:get_value`?

Reply:
Fixed. No idea why my brain when to proplists there. I try and use couch_util:get_value exclusively.